pages.it/linux/enum4linux.md

```diff
@@ -0,0 +1,24 @@
+# enum4linux
+
+> Strumento per ottenere informazioni da Windows e Samba da un sistema remoto.
+> Maggiori informazioni: <https://labs.portcullis.co.uk/tools/enum4linux/>.
+
+- Ottieni informazioni utilizzando tutti i metodi disponibili:
+
+`enum4linux -a {{host_remoto}}`
+
+- Ottieni informazioni utilizzando le credenziali fornite:
+
+`enum4linux -u {{nome_utente}} -p {{password}} {{host_remoto}}`
```

Comment:
Could you please translate password?

Reply:
@navarroaxel that is fine as is, "password" in Italian is still "password".